server/hotrod/src/test/java/org/infinispan/server/hotrod/HotRodMergeTest.java

```diff
@@ -73,6 +73,7 @@ protected void createCacheManagers() {
       waitForClusterToForm();
    }
 
+   @Test(enabled = false)
```

Comment:
Why are these disabled? There's no comment/description. Make sure you create a follow up JIRA to re-enable them or delete them if not relevant any more.